PMSCore/User.cs

```diff
@@ -37,8 +43,8 @@ public UserOperationResult CreateEntity(EntityType type, string entityId, string
                 break;
         }
 
-        AppendToReport($"Attempting to create entity [{entityId}]");
-        var result = EntityManager.GetInstance().CreateEntity(entityId, entityDescription, type);
+        AppendToReport($"Attempting to create {type.ToString()} [{entityId}]");
+        var result = StateManager.GetInstance().CreateEntity(type, entityId, entityDescription);
```

Comment:
AppendToReport($"Attempting to create entity [{entityId}]");

and 

AppendToReport($"Attempting to create {type.ToString()} [{entityId}]");

- maybe use just the 2nd so there are not more than 1 AppendToReport